persistence/src/commonMain/db_user/com/wire/kalium/persistence/Members.sq

```diff
@@ -31,10 +31,16 @@ SELECT * FROM Member
 JOIN Conversation ON Conversation.qualified_id = Member.conversation
 WHERE Member.user = ?;
 
+selectGroupConversationsByMember:
+SELECT * FROM Member
+JOIN Conversation ON Conversation.qualified_id = Member.conversation
+WHERE Member.user = ? AND Conversation.type = 'GROUP';
+
 updateMemberRole:
 UPDATE Member
 SET role = ?
 WHERE user = ? AND conversation = ?;
 
-deleteUserFromConversations:
-DELETE FROM Member WHERE user = ?;
+deleteUserFromConversation:
+DELETE FROM Member
+WHERE user = ? AND conversation = ?;
```

Comment:
I'm wondering if we can do it in one shot, doing a join in the `WHERE` clause, ie:

```
deleteUserFromConversation:
DELETE FROM Member
WHERE conversation IN (
    SELECT conversation FROM Member
    JOIN Conversation ON Conversation.qualified_id = Member.conversation
    WHERE Member.user = ? AND Conversation.type = 'GROUP';
) AND user = ? -- Probably this is not needed, since we filter before 
```

Reply:
Done :)